tests/drivers/uart/uart_async_api/testcase.yaml

```diff
@@ -20,9 +20,14 @@ tests:
     extra_args: DTC_OVERLAY_FILE="boards/nrf52840dk_nrf52840.overlay;boards/nrf_uart.overlay"
   drivers.uart.async_api.rtt:
     tags: drivers
-    filter: CONFIG_UART_CONSOLE and CONFIG_SERIAL_SUPPORT_ASYNC and CONFIG_HAS_SEGGER_RTT
+    filter: CONFIG_UART_CONSOLE and CONFIG_SERIAL_SUPPORT_ASYNC and CONFIG_HAS_SEGGER_RTT and not CONFIG_UART_MCUX_LPUART
     extra_args: DTC_OVERLAY_FILE=boards/segger_rtt.overlay
     extra_configs:
       - CONFIG_USE_SEGGER_RTT=y
       - CONFIG_UART_RTT=y
     build_only: true
+  drivers.uart.async_api.lpuart:
+    tags: drivers
+    filter: CONFIG_UART_CONSOLE and CONFIG_SERIAL_SUPPORT_ASYNC
```

Comment:
Since this is for testing LPUART, `and CONFIG_UART_MCUX_LPUART` is needed (or some other ways to target only device with LPUART).

Reply:
Will update the testcase accordingly